cpp/oneapi/dal/algo/decision_forest/common.hpp

```diff
@@ -257,61 +368,82 @@ class descriptor : public detail::descriptor_base<Task> {
     using method_t = Method;
     using task_t = Task;
 
+    /// @invariant {observations_per_tree_fraction > 0.0}
+    /// @invariant {observations_per_tree_fraction <= 1.0}
+    /// @remark default = 1.0
```

Comment:
All should be added to the corresponding getter, setter should not be documented. Please apply to other setters

Reply:
fixed